scripts/ci/5-run-tests.sh

```diff
@@ -78,6 +78,7 @@ if [ -z "$nose_args" ]; then
   --rednose \
   --with-timer \
   -v \
+  --stop \
```

Comment:
I don't think this is a good thing, in case multiple tests fails this mean a travis run for each test that fails instead of seeing them all at once

Reply:
This is not a hill I'm going to die defending, so I've removed it.